pkg/apiserver/filters/kubeapiserver.go

```diff
@@ -33,6 +35,8 @@ func WithKubeAPIServer(handler http.Handler, config *rest.Config, failed proxy.E
 			s := *req.URL
 			s.Host = kubernetes.Host
 			s.Scheme = kubernetes.Scheme
+			// remove cluster path
+			s.Path = strings.Replace(s.Path, fmt.Sprintf("/clusters/%s", info.Cluster), "", 1)
```

Comment:
Multiple cluster requests are dispatched to designated clusters before sent to kube-apiserver

Reply:
There is no difference between host cluster and other clusters. `/api/v1/clusters/host-cluster/nodes` Equivalent to `/api/v1/nodes`, even if it's not recommended.